packages/mask/.webpack/config.ts

```diff
@@ -120,10 +123,18 @@ export function createConfiguration(rawFlags: BuildFlags): Configuration {
                 { test: /(async-call|webextension).+\.js$/, enforce: 'pre', use: ['source-map-loader'] },
                 // TypeScript
                 {
-                    test: /\.tsx?$/,
+                    test: /\.(t|j)sx?$/,
+                    resolve: {
+                        fullySpecified: false, // disable the behaviour
```

Comment:
is it possible to not use this option? or limit it to a specific path by `test`.

Reply:
No. without this, it will not work for `@solana/wallet-adapter-*`.

Reply:
what about this:

```js
{
    test: /@solana\/wallet-adapter/,
    resolve: { fullySpecified: false }
}
```

And you should report it to the upstream so we can remove it when they fixed that.

Reply:
`@solana/wallet-adapter-*` need transpiling

> And you should report it to the upstream so we can remove it when they fixed that.

Some one already reported at https://github.com/solana-labs/wallet-adapter/issues/231

Reply:
OK, so please split it as a separate rule and don't touch our default rule

Reply:
https://github.com/solana-labs/wallet-adapter/issues/231#issuecomment-1003465379

^ the packages have been updated, this issue may be fixed. The packages no longer require transpilation because they ship ESM and CJS versions now.

Reply:
@jordansexton Wow, thanks.

Reply:
Looks like it didn't fix. Still get same errors without those config updates.

![image](https://user-images.githubusercontent.com/1141198/147928780-df7c4ab9-a525-4c18-88d4-075f0c1c780d.png)

Reply:
I'm not sure what your build config is, but please first check out the starter projects in the WalletAdapter repo, since they are confirmed to be working. Please open an issue with a link to a reproduction if that doesn't help.